### ISSUE TITLE
VxMark*: when navigating to and from the review screen, keep your place in the list

### ISSUE THREAD

**Issue by adghayes** [votingworks/vxsuite#8404]
When navigating with the ATI, if you review "Ballot Proposition C" and then return to it, the focus should be on "Ballot Proposition C" again. We might have to read some variant of the instructions here.

When navigating visually, it would be helpful if the last reviewed contest scrolled into focus.

**Comment by adghayes** [comment]
I took a little look into this and found it to be not as simple as I'd like, also there's some audio UX to think about.

It's easy enough to pass the contest ID as a URL param and then use an effect to set focus (and scroll) to the contest you were just reviewing. The audio impact of that would be to skip the review page instructions and repeat the summary of the contest. I find this a little confusing - you just navigated away from the contest and it's suddenly reading you the contest information again. On the other hand, I don't think repeating the full instructions is useful. My idea would be that it reads the header, "Review Your Votes", to orient the user to the review page, and then it can start reading the summary of the current contest. But having that level of control over audio would, I think, require some additions to the screen reader.
